### PR TITLE
Created a presentational component that lists markdown files.

### PR DESCRIPTION
**Pull request by Zilula**
Created a functional component that displays a list of available markdown files from our application`s store.

### CODE REVIEW

File: src/components/SavedMarkdownList.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+export default function MarkdownList({ markdownList }) {
```
Comment (mehtaphysical) [inline src/components/SavedMarkdownList.js]: If you only are going to display the titles, why no just pass titles instead of the entire file.

Comment (mehtaphysical) [reply src/components/SavedMarkdownList.js]: You would then create a `getTitles` selector

Comment (Zilula) [reply src/components/SavedMarkdownList.js]: > You would then create a `getTitles` selector

We will add a selector to get a list of titles in the container component work.
Thank you,
Lance

**Review by MariahAdams — APPROVED** [review]
Thumbs up!

**Pull request merged**